docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 breathe==4.22.1
 sphinx-click==2.5.0
 pydata-sphinx-theme
+sphinx-inline-tabs
```

Comment:
`sphinx-tabs` are not grouped by default, the author must explicitly declare them as grouped:

https://github.com/executablebooks/sphinx-tabs

`sphinx-inline-tabs` have this grouping behavior as a feature.

> Synchronisation: Tabs with the same label all switch with a single click.

Reply:
Sorry I didn't get your point.
Do you want to disable the synchronization?

Reply:
I just think that content swinging to top and bottom are a bit distracting.